ts/features/cieLogin/components/screens/CieLoginConfigScreen.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useCallback } from "react";
+import {
+  Button,
+  InputAccessoryView,
+  Platform,
+  SafeAreaView,
+  ScrollView,
+  View
+} from "react-native";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import { ContentWrapper } from "../../../../components/core/ContentWrapper";
+import CiePinpad from "../../../../components/CiePinpad";
+import { IOColors } from "../../../../components/core/variables/IOColors";
+import CieLoginConfigScreenContent from "./CieLoginConfigScreenContent";
+
+type PinViewProps = {
+  pin: string;
+  setPin: (pin: string) => void;
+  onSubmit: (pin: string) => void;
+};
+const PinView = (props: PinViewProps) => (
+  <>
+    <CiePinpad
+      pin={props.pin}
+      pinLength={6}
+      onPinChanged={props.setPin}
+      InputAccessoryViewID="pinInputId"
+      onSubmit={props.onSubmit}
+    />
+    {Platform.OS === "ios" && (
+      <InputAccessoryView
+        backgroundColor={IOColors.greyUltraLight}
+        nativeID={"pinInputId"}
+      >
+        <View
+          style={[
+            IOStyles.flex,
+            IOStyles.horizontalContentPadding,
+            { justifyContent: "flex-end", flexDirection: "row" }
+          ]}
+        >
+          <Button
+            onPress={() => props.onSubmit(props.pin)}
+            title="Sblocca"
+            accessibilityLabel="Premi qui per sbloccare"
+          />
+        </View>
+      </InputAccessoryView>
+    )}
+  </>
+);
+
+const CieLoginConfigScreen = () => {
+  const [locked, setLockec] = React.useState(true);
+  const [pin, setPin] = React.useState("");
+
+  const onSubmit = useCallback((pin: string) => {
+    // constant day containig the current day in the format YYMMDD
+    const day = new Date().toISOString().slice(2, 10).replace(/-/g, "");
+    if (pin === day) {
+      setLockec(false);
```

Comment:
```suggestion
      setLocked(false);
```

Reply:
Fixed in d43b36077b5af300023e094adc9ddd690729bf3c . Good catch! 💪